bundlegen/src/main/java/com/citrix/microapps/bundlegen/bundles/BundlesLoader.java

```diff
@@ -269,6 +273,16 @@ private static boolean isTranslationValid(TemplateFile template, ModelTranslatio
                         String.format("Missing the translation checksum %s", fileName))) : Optional.empty();
     }
 
+    static Optional<ValidationException> isTimestampValid(String field, String timestamp) {
```

Comment:
The "is" in the method's name suggests it returns 'boolean', which it doesn't. I'd rename this method to 'validateDeprecatedDate', and the 'field' parameter is probably unnecessary, this code is currently used to validate "deprecatedDate" only, if I'm not mistaken.

Reply:
Method renamed and an unnecessary  parameter removed